project/Dependencies.scala

```diff
@@ -10,7 +10,11 @@ object Dependencies {
   // sbt modules
   private val ioVersion = "1.2.1"
   private val utilVersion = "1.2.2"
-  private val lmVersion = "1.2.1"
+  private val lmVersion =
+    sys.props.get("sbt.librarymanagement.version") match {
```

Comment:
I recently added build-time flag for fatal warnings and I used `sbt.build` prefix.
Could we rename this to `sbt.build.lm.version`?